specification/storagesync/resource-manager/readme.ruby.md

```diff
@@ -12,7 +12,6 @@ azure-arm: true
 
 ``` yaml $(ruby) && $(multiapi)
 batch:
-  - tag: package-2018-07-01
```

Comment:
It is still appropriate to include this tag as part of `multiapi` languages.

Reply:
Incorporated